apps/rush-lib/src/logic/installManager/WorkspaceInstallManager.ts

```diff
@@ -531,7 +531,11 @@ export class WorkspaceInstallManager extends BaseInstallManager {
     }
 
     const localDependencyProjectNames: Set<string> = new Set<string>(
-      project.localDependencyProjects.map((x) => x.packageName)
+      (function* (deps: Iterable<RushConfigurationProject>): Iterable<string> {
+        for (const dep of deps) {
+          yield dep.packageName;
+        }
+      })(project.localDependencyProjectSet)
```

Comment:
`project.localDependencyProjects.map((x) => x.packageName)` really is more intuitive IMO

Reply:
Well, `[...project.localDependencyProjects].map(x => x.packageName)` is the closest equivalent, but I can do it, it's not a perf bottleneck.

Reply:
Switched to using the array and map. Wonder if it might be worth extending `Set` with `.map()` for ergonomics.